### PR TITLE
Build the table layout

### PR DESCRIPTION
**Pull request by ClementChaumel**
Fixes canonical-web-and-design/commercial-squad#250

## Done

- Styled the table to match the designs
- the cats are gone :sob: 
- 
## QA

- go to https://ubuntu-com-10438.demos.haus//cube/microcerts
- see that both tables look pretty much the same

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/250

## Screenshots

image

### CODE REVIEW

File: static/js/src/cube/components/MicrocertificationsTable/MicrocertificationsTable.tsx [hunk 1]
```diff
@@ -32,10 +33,13 @@ const TableView = () => {
           columns: [
             {
               content: "1",
-              role: "rowheader",
+              className: "u-text--muted",
             },
             {
-              content: <img src="http://placekitten.com/45/45" />,
+              content: (
+                <img src="https://assets.ubuntu.com/v1/9ef4a092-Architecture.svg" />
```
Comment (petermakowski) [inline static/js/src/cube/components/MicrocertificationsTable/MicrocertificationsTable.tsx:40]: Missing alt tag. What about we match what's already in the existing section? 
image


```suggestion
                <img src="https://assets.ubuntu.com/v1/9ef4a092-Architecture.svg" alt="Badge for Ubuntu System Architecture" />
```

File: static/js/src/cube/components/MicrocertificationsTable/MicrocertificationsTable.tsx [hunk 2]
```diff
@@ -56,10 +67,13 @@ const TableView = () => {
           columns: [
             {
               content: "2",
-              role: "rowheader",
+              className: "u-text--muted",
             },
             {
-              content: <img src="http://placekitten.com/44/46" />,
+              content: (
+                <img src="https://assets.ubuntu.com/v1/c19e9931-Packages.svg" />
```
Comment (petermakowski) [inline static/js/src/cube/components/MicrocertificationsTable/MicrocertificationsTable.tsx:74]:
```suggestion
                <img src="https://assets.ubuntu.com/v1/c19e9931-Packages.svg" alt="Badge for Managing Packages in Ubuntu" />
```

**Comment by petermakowski** [comment]
I think you may need to add aria-labels to table cells in order for the mobile view to work. I had a similar issue: https://github.com/canonical-web-and-design/ubuntu.com/pull/10445/files#diff-a7b102cf15680e8a30b7c4270aca5d759e4959eff6fb32c0c6f156761cf6e6a8R140

image

**Comment by ClementChaumel** [comment]
> I think you may need to add aria-labels to table cells in order for the mobile view to work. I had a similar issue: https://github.com/canonical-web-and-design/ubuntu.com/pull/10445/files#diff-a7b102cf15680e8a30b7c4270aca5d759e4959eff6fb32c0c6f156761cf6e6a8R140
> 
> image

Thanks I'll update it

File: static/js/src/cube/components/MicrocertificationsTable/MicrocertificationsTable.tsx (hunk 1 above)
Comment (sowasred2012) [reply static/js/src/cube/components/MicrocertificationsTable/MicrocertificationsTable.tsx:40]: I'd counter that suggestion with an empty alt tag - in this context I'm not sure the image adds anything critical that someone using assistive technology couldn't glean from the title of the module:

```suggestion
                <img src="https://assets.ubuntu.com/v1/9ef4a092-Architecture.svg" alt="" />
```

From [WebAIM](https://webaim.org/techniques/alttext/):

> If the content that the image conveys is presented within text in the surrounding context of the image, then an empty alt attribute may suffice.

Comment (ClementChaumel) [reply static/js/src/cube/components/MicrocertificationsTable/MicrocertificationsTable.tsx:40]: Yeah I know the content of the rows is not correct yet. I was planning to do that as part of this ticket : https://app.zenhub.com/workspaces/-commercial-squad-60910a7b9f3626000eff21c2/issues/canonical-web-and-design/commercial-squad/251

Comment (petermakowski) [reply static/js/src/cube/components/MicrocertificationsTable/MicrocertificationsTable.tsx:40]: @sowasred2012  Yup, empty alt tag totally works, too. As long as an image alt tag is handled in _some_ way.

**Comment by petermakowski** [comment]
"Module number" and "badge" labels seem to be missing. Is this intentional? 
image

**Comment by ClementChaumel** [comment]
> "Module number" and "badge" labels seem to be missing. Is this intentional?
> image

Yes, I omitted them because they felt redundant? @wgx @elioqoshi should I go to UX prison? What do you think?

**Comment by wgx** [comment]
@ClementChaumel you may escape UX prison and proceed :) 👍🏽

**Pull request merged**